Dockerfile

```diff
@@ -4,6 +4,15 @@ WORKDIR /app
 
 COPY ./client .
 
+ENV PYTHONUNBUFFERED=1
+RUN apk add --update --no-cache \
+      g++ \
+      make \
+      python3 && \
+  ln -sf python3 /usr/bin/python && \
+  python3 -m ensurepip && \
+  pip3 install --no-cache --upgrade pip setuptools
+
 RUN yarn
```

Comment:
Using yarn install with `--ignore-scripts` appears to do the job too. Python is needed to run a script from deasync package. I tried running the Docker image without it and it doesn't appear to be needed.